lib/app/utils/startup_tasks.rb

```diff
@@ -8,6 +8,20 @@
 module Inferno
   module StartupTasks
     class << self
+
+      CONFORMANCE_RESOURCE_TYPES = [
+        'ImplementationGuide',
+        'StructureDefinition',
+        'ValueSet',
+        'ConceptMap',
+        'CodeSystem',
+        'CapabilityStatement',
```

Comment:
I don't know what `CapabilityStatement`, `OperationDefinition`, `SearchParameter`, or `CompartmentDefinition` would play in resource validation.